cluster/gce/util.sh

```diff
@@ -660,7 +660,17 @@ function create-network() {
       --project "${PROJECT}" \
       --network "${NETWORK}" \
       --source-ranges "10.0.0.0/8" \
-      --allow "tcp:1-65535,udp:1-65535,icmp" &
+      --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
```

Comment:
This rule means that accessing this port on any machine, including regular nodes, is not allowed. I think we should only limit this to the master. WDYT?